### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by kayh1105** [fjykTec/ModernWMS#11]
![Screenshot 2023-03-22 111244](https://user-images.githubusercontent.com/103476694/226794229-dc321a06-fb89-476e-8092-c137e82b40bd.jpg)

It got this error when I try to run the .dll. this at the last step

**Comment by leucoon** [comment]
Replace the last command with the following
 `cd /ModernWMS/backen/  && dotnet ModernWMS.dll --urls http://0.0.0.0:20011`

**Comment by kayh1105** [comment]
thanks for the reply. yes noticed that, and changed. But seems like the backend still not running even though I've open that particular port. website still can't use
![Untitled](https://user-images.githubusercontent.com/103476694/229016202-11bd2e49-c303-4ad1-bb84-1236b7c2de7a.png)

**Comment by kayh1105** [comment]
I've followed the steps. Now the backend running when I checked http://ipaddress:20011 and try out the api. seems like database already connected correctly. But when I go to http://ipadress still the same. Cannot read the user and cannot register new too

**Comment by leucoon** [comment]
var your screenshot of file [.env.production] , you have the right modify. Please check whether the compiled front end is accurately overwritten to the nginx directory "/etc/nginx/html/" .
if not, please retry the following
`cd /tmp/ModernWMS-master/frontend/ `
`yarn && yarn build `
`cp -rf  /tmp/ModernWMS-master/frontend/dist/* /etc/nginx/html/`

**Comment by kayh1105** [comment]
> Through the screenshot of the file [.env.production] provided by you, you have modified it correctly. Please check whether the compiled front end is accurately overwritten to the nginx directory "/etc/nginx/html/" . if not, please retry the following `cd /tmp/ModernWMS-master/frontend/ ` `yarn && yarn build ` `cp -rf /tmp/ModernWMS-master/frontend/dist/* /etc/nginx/html/`

Actually I've done it all for few times already. Whenever I built new frontend, I'll check on "/etc/nginx/html/" folder. They're copied and updated  whenever I executed the commands. That's why I'm not sure what else I'm missing that the ip address not changing.

**Comment by leucoon** [comment]
Have you cleared the browser's cache after recompiling and overwritten to the nginx directory ?

**Comment by kayh1105** [comment]
yes, cleared cache every time I rebuild. I even redo everything back for frontend in case I miss any step. But still request from http:127.0.0.1
![1](https://user-images.githubusercontent.com/103476694/231078903-fede669e-4234-41ad-aaad-dec6ade02bf5.jpg)
![2](https://user-images.githubusercontent.com/103476694/231078909-f99fdbcc-3d76-46d6-8305-abdef5fe0786.jpg)

**Comment by leucoon** [comment]
It is recommended that you find a new server and reinstall it.
